api/stats.rb

```diff
@@ -0,0 +1,41 @@
+get "#{APIPREFIX}/stats" do # retrieve threads by course
+  from_time = Date.today.to_time - (params["limit_days"] or 1.months)
+  response_data = {}
+  request_user = nil
+  if params["user_id"]
```

Comment:
There should always be a `request_user` because the user-specific data that this returns wouldn't make sense.